src/cultionet/models/lightning.py

```diff
@@ -315,66 +315,53 @@ def scale_logits(x: torch.Tensor, t: torch.Tensor) -> torch.Tensor:
 class TemperatureScaling(LightningModule):
     def __init__(
         self,
-        cultionet_model: CultioNet = None,
-        learning_rate: float = 1e-4,
+        num_classes: int = 2,
+        learning_rate_refine: float = 1e-3,
         learning_rate_lbfgs: float = 0.01,
         weight_decay: float = 0.01,
         eps: float = 1e-4,
         max_iter: float = 25,
         edge_class: int = 2,
-        class_counts: T.Optional[torch.Tensor] = None
+        class_counts: T.Optional[torch.Tensor] = None,
+        cultionet_ckpt: T.Optional[T.Union[Path, str]] = None
     ):
         super(TemperatureScaling, self).__init__()
 
-        self.crop_temperature = torch.nn.Parameter(torch.ones(1))
+        self.save_hyperparameters()
```

Comment:
ok, so you add this step to  saved arguments in the checkpoint file

Reply:
Correct, `save_hyperparameters` is a PyTorch Lightning method.